### PR TITLE
[CNTT TSC] Overall Technical Roadmap and Dependencies

### LINKED ISSUE #968: [TSC] Create and sign off technical Roadmap
Agree on Technical Roadmap and sign it off.

### LINKED ISSUE #1158: [RM Ch1.9]: Roadmap chapter is obsolete

### LINKED ISSUE #1177: [TSC] Agree on Technical Releases relation to WSs and with other communities
As Discussed in TSC, we need to agree on WSs releases (versions) within CNTT itself and with other relevant communities.

### PR DESCRIPTION
**Pull request by rabi-abdel**
Fixes #968 
Fixes #1177 
Fixes #1158

### CODE REVIEW

**Comment by rabi-abdel** [comment]
@mf4716 @scottsteinbrueck based on the discussions we had during TSC today, let us draft something and use as a starting point. I will draft something in the coming few days.

File: doc/tech/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Roadmap
+
+<a name="1.1"></a>
+### High Level Roadmap
+<p align="center"><img src="./figures/cntt_roadmap.png" alt="scope" title="Roadmap" width="100%"/></p>
+<p align="center"><b>Figure 1:</b> CNTT Technical Specification Roadmap</p>
+
+<a name="1.1"></a>
+### Detailed Roadmap
+Please refer to
```
Comment (tomkivlin) [inline doc/tech/roadmap.md]: to what? 😉

Comment (rabiabdel) [reply doc/tech/roadmap.md]: link should be valid now!

File: doc/tech/roadmap.md [hunk 2]
```diff
@@ -46,7 +48,17 @@ A Release is:
 Please refer to individual [release plan](./release_notes/) and features for detailed roadmap.
 
 <a name="2.3"></a>
-### Milestones
+### Detailed Milestones
+| Review | Milestone | Description | Activities | Comments |
+|--------|-----------|--------------|------------|---------|
+| Kick-Off | M0 | The goal of the release Kick-Off is to open the innovation platform for the intent to participate in the CNTT release. Release Kick-Off review takes place for each releases. | Identity a list of features and functionality will be documented and developed. | |
```
Comment (pgoyal01) [inline doc/tech/roadmap.md]: what is the "innovation platform"?

The sentence "The goal of the release Kick-Off is to open the innovation platform for the intent to participate in the CNTT release" -- goal and intent -- can be simplified and the goal also has to be more than just indicating an intent to participate.

Typo: 
"Release Kick-Off review takes place for each releases" -- singular "release"
"Identity a list of features.." --- "Identify..."

File: doc/tech/roadmap.md [hunk 3]
```diff
@@ -46,7 +48,17 @@ A Release is:
...
+| Planning | M1 | The goal of the Release Planning review is to ensure the plan is complete, sufficient, and aligned with release milestones. All people resources are identified, documented and committed. | Identify what is in or out of scope. During the development phase, it helps reduce the discussion. Highlight the issues to TSC. | |
```
Comment (pgoyal01) [inline doc/tech/roadmap.md]: Is it "Release Planning" or "Release Planning review"? If a review then where is the release plan developed?
Sentence issues

File: doc/tech/roadmap.md [hunk 4]
```diff
@@ -46,7 +48,17 @@ A Release is:
...
+| Issue Logging Freeze | M2 | The goal of the Issue Logging Freeze is to mark the end of adding new issues in the Release. |  All the project leads authorise the issues are correctly labelled, mapped to the corresponding project and milestone, etc., | |
```
Comment (pgoyal01) [inline doc/tech/roadmap.md]: Instead of "Issue logging" can we use "Change Freeze" where requested changes are logged into Github as issues; I think post-M2 issues related to erros are still allowed.

File: doc/tech/roadmap.md (hunk 2 above)
Comment (prabhubalan) [reply doc/tech/roadmap.md]: @pgoyal01 - Thanks for correcting the typo. This is indeed an innovation platform but there is no need to explain or define what is innovation platform here. Let me see if I can better reword it.

File: doc/tech/roadmap.md (hunk 3 above)
Comment (prabhubalan) [reply doc/tech/roadmap.md]: After M0 and the day before M1 is the release planning. Review happens on the cut-off date as per the agreed M1 date to see what goes in and what goes out as per the delivery timeline in the release process. Hence I have defined release planning review.  M0 to M1 is Release Planning and M1 cutoff date is the day for Release Planning Review.

File: doc/tech/roadmap.md (hunk 4 above)
Comment (prabhubalan) [reply doc/tech/roadmap.md]: @pgoyal01  - I will better reword it to "Github Issue Logging" rather than "Issue Logging" or "Change Freeze". Again question might raise “what is change freeze”. The issues are logged not only to change, also to add, remove

**Comment by rabi-abdel** [comment]
@tomkivlin can you please review/approve

**Comment by rabi-abdel** [comment]
@prabhubalan can you please resolve the PR? as it is showing conflict
